src/JSInterop/Microsoft.JSInterop.JS/src/src/Microsoft.JSInterop.ts

```diff
@@ -154,13 +154,14 @@ export module DotNet {
    * Parses the given JSON string using revivers to restore args passed from .NET to JS.
    *
    * @param json The JSON stirng to parse.
+   * @param byteArrays The byte array data to revive.
    */
-  function parseJsonWithRevivers(json: string): any {
+  function parseJsonWithRevivers(json: string, byteArrays?: Uint8Array[] | null): any {
     return json ? JSON.parse(json, (key, initialValue) => {
       // Invoke each reviver in order, passing the output from the previous reviver,
       // so that each one gets a chance to transform the value
       return jsonRevivers.reduce(
-        (latestValue, reviver) => reviver(key, latestValue),
+        (latestValue, reviver) => reviver(key, latestValue, byteArrays === undefined ? null : byteArrays),
```

Comment:
I don't think it's necessary to convert `undefined` to `null` here since our logic can treat `undefined` the same as `null`.

Reply:
Normally I wouldn't bother about such fine-tuning but JSON revivers are called for every single object nested in the whole graph on every JS interop call, so they run the risk of being a bottleneck.